test/providers.test.js

```diff
@@ -13,6 +13,18 @@ test('json rpc fetch block info', async () => {
     const provider = new nearlib.providers.JsonRpcProvider(config.nodeUrl);
     let response = await provider.block(1);
     expect(response.header.height).toEqual(1);
+    let sameBlock = await provider.block(response.header.hash);
+    expect(sameBlock.header.height).toEqual(1);
+});
+
+test('json rpc fetch chunk info', async () => {
+    const config = Object.assign(require('./config')(process.env.NODE_ENV || 'test'));
```

Comment:
can we extract this into helper or `before`? Seems like every test is loading config and creating provider with it.

Reply:
Sure, I will fix it.